ribs/emitters/_gaussian_emitter.py

```diff
@@ -44,6 +44,7 @@ def __init__(self,
                  archive,
                  x0,
                  sigma0,
```

Comment:
I actually think it makes more sense if `sigma` is the parameter that people pass in, and `sigma0` is the one that can additionally be passed in, i.e. you would usually sample with `sigma` variance when the archive is empty, but if `sigma0` is passed in, then you sample with `sigma0` variance when the archive is empty.